test/hotspot/jtreg/vmTestbase/gc/gctests/StringInternGC/StringInternGC.java

```diff
@@ -46,13 +46,17 @@
  * String pool should not overflow.
  */
 public class StringInternGC extends ThreadedGCTest {
-        private int maxLength = 1000;
+        private int maxLength = 1000; // Maximum length to add per operation.
+        private int maxTotalLength = 128 * 1024; // Random total maximum length of the string.
```

Comment:
Please remove the "random" word - that has only been for you to indicate that this value is random in the discussion